app/build.gradle

```diff
@@ -253,7 +189,6 @@ apollo {
 repositories {
     // This is the only reference we cannot remove yet as it has not been migrated to mavenCentral() or google()
```

Comment:
This comments could also be removed as it was an explanation around why we were keeping jcenter() repository